src/build.sh

```diff
@@ -7,8 +7,13 @@ SRC=""
 #MODE="PROFILE"
 #MODE="MEMCHECK"
 MODE="RELEASE"
+VERSION_FILE="$(cd "$(dirname "$1")" || exit; pwd)/$(basename version.inc)"
```

Comment:
Hmm OK fair.

Could you make this clearer by pulling it out into a function, like in http://stackoverflow.com/a/21188136/3191, and then commenting that we need the absolute path because different parts of this script manipulate it from different directories?